tests/rest/client/test_room_access_rules.py

```diff
@@ -670,6 +720,126 @@ def test_revoke_3pid_invite_direct(self):
             tok=self.tok,
         )
 
+    def test_check_event_allowed(self):
+        """Tests that AccessRules.check_event_allowed behaves accordingly"""
```

Comment:
```suggestion
        """Tests that RoomAccessRules.check_event_allowed behaves accordingly"""
```

Reply:
Also, might be worth summarising what "behaving accordingly" means in this context.